pages/events/index.tsx

```diff
@@ -35,7 +26,7 @@ export default function EventsIndexPage(
     new URLSearchParams(router.asPath.split(/\?/)[1]).get("past") === "1";
 
   return (
-    <HydrationBoundary state={props.dehydratedState}>
```

Comment:
we should look into HydrationBoundary if it is still relevant

Reply:
added hydration boundary and prefetching of query in static props as previously implemented